src/main/java/com/dezorganizacja/bridge/model/Game.java

```diff
@@ -0,0 +1,32 @@
+package com.dezorganizacja.bridge.model;
+
+import java.util.ArrayList;
+
+/**
+ * Created by wiktortendera on 15/04/15.
+ */
+public class Game {
+    ArrayList<Player> players;
+    private Deck deck;
+    public Game(Player player1, Player player2, Player player3, Player player4) {
+        players = new ArrayList<>();
+        players.add(player1);
+        players.add(player2);
+        players.add(player3);
+        players.add(player4);
+    }
+
+    public Player getPlayer(int index) {
+        if(index > 4 || index <=0)
```

Comment:
[detal] nie wiem czy nie lepiej a) zostawić liscie walidacji czy indeks jest dobry b) rzucić jakiś własny wyjątek. Wydaje mi się ze a) lepsze bo tak jak teraz to teoretycznie wymaga za kazdym razem sprawdzania czy player nie jest przypadkiem nullem co moze skutkować NullPointerException w jakimś dziwnym miejscu. No ale jak mówię detal i nie wiem. @gswirski ?